docs/how_to/introduction_to_deep_learning_guide.md

```diff
@@ -0,0 +1,49 @@
+---
+align: center
+---
+Artificial Intelligence Venn Diagram
+```
+
+Machine Learning is a field within Artificial Intelligence (AI) that focuses on an algorithm that can learn from experience or data and automatically improve outcomes. These algorithms can adapt to new circumstances, unlike explicitly programmed programs. These algorithms detect patterns in the underlying data and modify an underlying model to extrapolate to new situations. Figure 1 illustrates the interrelated relationship of each algorithm field within AI.
+
+Within ML, this guide primarily focuses on supervised learning algorithms and the training data consisting of the model inputs and desired outputs during the training phase. The training algorithm learns the underlying model parameters through an iterative optimization process. During the evaluation phase, the model topology with the trained model parameters from the learned agent is evaluated based on its performance against new data.
+
+## What Is Deep Learning?
+
+Deep Learning algorithms are a class of ML algorithms inspired by biological neural networks and comprise multiple levels of superficial component layers. The earliest component classifier is the perceptron, a binary classifier comprising a linear, fully connected stage with a non-linear decision stage. Modern DL component layers include widely accepted layers such as:
+
+- Convolution
+- Activation
+- Fully connected, long short-term memory
+- Custom layers designed by application developers
+
+The training of a DL network uses large amounts of data. DL algorithms use Deep Neural Networks to access, explore, and analyze vast sets of information—such as all the music files on streaming platforms that make ongoing suggestions based on the tastes of a specific user. The input—whether an image, a new article, or a song—is evaluated in its raw or untagged form with minimal transformation. This unsupervised training process is sometimes called representation learning. During training, the DL algorithm progressively learns from the data to improve the accuracy of its conclusions (also known as inference). Table 1 provides DL applications and their usage.
+
+#### Table 1. Common Examples of Deep Learning
+
+:::{table} Common Examples of Deep Learning
+:name: CommonExamplesofDeepLearning
+:widths: auto
+| Deep Learning Examples | Usage |
+|------------------------|-------|
+| Autonomous Driving | Combining deep data (maps, satellite traffic images, weather reports, a user's accumulated preferences), real-time sensor input from the environment (a deer on the road, a swerving driver), and compute power to make decisions (slow down, turn the steering wheel). |
+| Medical | Cancer research—for example, learning to detect melanoma in photos. |
+| Smart Home | Smart speakers use intelligent personal assistants and voice-recognition algorithms to comprehend and respond to unique users' verbal requests. |
+
+## Why Use AMD GPUs for Deep Learning?
+
```

Comment:
This content belongs in amd.com websites outside of documentation. The proper connection is to the marketing team is  being made offline.